backends/bmv2/common/control.cpp

```diff
@@ -352,7 +352,9 @@ ControlConverter::convertTable(const CFG::TableNode* node,
             // 1) if there is at least one RANGE field, then the table is RANGE
             // 2) if there is at least one TERNARY or OPTIONAL field, then the table is TERNARY
             // 3) if there is a LPM field, then the table is LPM
-            // 4) otherwise the table is EXACT
+            // 4) if there is a LPM field, then the table is ternary or EXACT
```

Comment:
cases 3) and 4) seems at odds with each other:
> // 3) if there is a LPM field, then the table is LPM
// 4) if there is a LPM field, then the table is ternary or EXACT

Besides, I think the rules are correct and do not need to be changed.